amy/templates/emails/scheduled_email_detail.html

```diff
@@ -134,30 +126,31 @@ <h1>
       {% endif %}
     </td>
   </tr>
-  <tr>
-    <th>History:<br><small>(newest to oldest)</small></th>
-    <td>
-      <table class="table">
-        <tr>
-          <th>Timestamp</th>
-          <th>State before</th>
-          <th>State after</th>
-          <th>Details</th>
-          <th>ID</th>
-        </tr>
+</table>
 
-        {% for log in log_entries %}
-        <tr>
-          <td>{{ log.created_at }}</td>
-          <td>{{ log.state_before }}</td>
-          <td>{{ log.state_after }}</td>
-          <td><code>{{ log.details }}</code></td>
-          <td>{{ log.id }}</td>
-        </tr>
-        {% endfor %}
-      </table>
-    </td>
-  </tr>
+<table class="table table-striped">
+  <caption>History<br> <small>(newest to oldest)</small></caption>
```

Comment:
I found that this text gets slightly lost on the page, so suggestion to make it a bit more obvious
```suggestion
  <caption><h2>History</h2> <small>(newest to oldest)</small></caption>
```